app-prod.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: "nais.io/v1alpha1"
 kind: "Application"
 metadata:
   name: familie-integrasjoner
-  namespace: default
+  namespace: teamfamilie
```

Comment:
Nå virker det som att det finnes 2 yamlfiler for prod med namespace: teamfamilie?